cmd/batch-handlers.go

```diff
@@ -833,21 +834,35 @@ func (ri *batchJobInfo) clone() *batchJobInfo {
 		ObjectsFailed:    ri.ObjectsFailed,
 		BytesTransferred: ri.BytesTransferred,
 		BytesFailed:      ri.BytesFailed,
+		Attempts:         ri.Attempts,
 	}
 }
 
-func (ri *batchJobInfo) countItem(size int64, dmarker, success bool) {
+func (ri *batchJobInfo) countItem(size int64, dmarker, success bool, attempt int) {
 	if ri == nil {
 		return
 	}
+	ri.Attempts++
```

Comment:
ri.Attempts does not seem to be used here, are we trying to send this information to the user in the future ? also it seems to me it is more useful if we do this:

```
if attempt > 0 {
ri.Attempts++
}
```

Reply:
Attempt should always be >0, if not it is a mistake by the caller. 

Yes. It is just to log the total number of attempts.